src/storage/v2/edge_accessor.hpp

```diff
@@ -63,6 +63,9 @@ class EdgeAccessor final {
   /// @throw std::bad_alloc
   Result<bool> InitProperties(const std::map<storage::PropertyId, storage::PropertyValue> &properties);
 
+  Result<std::vector<std::tuple<PropertyId, PropertyValue, PropertyValue>>> UpdateProperties(
+      std::map<storage::PropertyId, storage::PropertyValue> &properties) const;
```

Comment:
Just a note, no action needed: I saw that Google preferred for a long time pointers when modifying arguments and they used references only when they could be const. But they changed that so don't bother please

Reply:
I heard about that too, forgot about it when coding, but remembered it now. Cool stuff will remember that for next time.